airflow/kubernetes/pod_launcher.py

```diff
@@ -178,12 +178,17 @@ def monitor_pod(self, pod, get_logs):
         if self.extract_xcom:
             result = self._extract_xcom(pod)
```

Comment:
not yet handled is the case of _extract_xcom throws an error, for example return.json is not written by user in KPO